projects/g++/Menge/MengeVis/CMakeLists.txt

```diff
@@ -56,6 +56,6 @@ add_library(
 	${source_files}
 )
 
-target_link_libraries ( mengeVis  dl tinyxml ${OPENGL_LIBRARIES} ${SDL_LIBRARY} ${PNG_LIBRARY}  ${SDLIMAGE_LIBRARY}  ${SDLTTF_LIBRARY} )
+target_link_libraries ( mengeVis mengeCore dl tinyxml ${OPENGL_LIBRARIES} ${SDL_LIBRARY} ${PNG_LIBRARY}  ${SDLIMAGE_LIBRARY}  ${SDLTTF_LIBRARY} )
```

Comment:
This seems to be mandatory. I don't understand how you could get this compiled under under windows